draft-davidson-pp-protocol.md

```diff
@@ -815,223 +589,105 @@ and is necessary for achieving the unforgeability requirement.
-
-The VOPRF ciphersuite {{I-D.irtf-cfrg-voprf}} used determines
-the member functions and prime-order group used by the protocol. We
-detail a number of specific conventions here that we use for interacting
-with the specific ciphersuite.
-
-### Ciphersuites
-
-Let `F` denote a generic VOPRF API function as detailed in
-{{I-D.irtf-cfrg-voprf}} (Section TODO), and let `ciph` denote the
-ciphersuite that is used for instantiating the VOPRF. In this document,
-we explicitly write `ciph.F` to show that `F` is explicitly evaluated
-with respect to `ciph`.
-
-In addition, we define the following member functions associated with
-the ciphersuite.
-
-- `recover_ciphersuite_from_id(id)`: Takes a string identifier `id` as
-  input, and outputs a VOPRF ciphersuite. Returns `null` if `id` is not
-  recognized.
-- `group()`: Returns the prime-order group associated with the
-  ciphersuite.
-- `H1()`: The function `H1()` defined in {{I-D.irtf-cfrg-voprf}}
-  (Section TODO). This function allows deterministically mapping
-  arbitrary bytes to a random element of the group. In the elliptic
-  curve setting, this is achieved using the functions defined in
-  {{I-D.irtf-cfrg-hash-to-curve}}.
-
-### Prime-order group conventions
-
-We detail a few functions that are required of the prime-order group
-`GG` used by the VOPRF in {{I-D.irtf-cfrg-voprf}}.
-
-Let `p` be the order of the Galois field `GF(p)` associated with the
-group `GG`. We expose the following functions associated with `GG`.
-
-- `GG.generator()`: Returns the fixed generator associated with the
-  group `GG`.
-- `GG.scalar_field()`: Provides access to the field `GF(p)`.
-- `GG.scalar_field().random()`: Samples a scalar uniformly at random
-  from GF(p). This can be done by sampling a random sequence of bytes
-  that produce a scalar `r`, where `r < p` is satisfied (via
-  rejection-sampling).
-
-We also use the following functions for transitioning between different
-data types.
-
-- `as_bytes()`: For a scalar element of `GG.scalar_field()`, or an
-  element of `GG`; the `as_bytes()` functions serializes the element
-  into bytes and returns this array as output.
-- `as_scalar()`: Interprets a sequence of bytes as a scalar value in
-  `GG.scalar_field()`. For an array of byte arrays, we define the
-  function `as_scalars()` to individually deserialize each of the
-  individual byte arrays into a scalar and output a new array containing
-  each scalar value.
-- `as_element()`: Interprets a sequence of bytes as a group element in
-  `GG`. For an array of byte arrays, we define the function
-  `as_elements()` to individually deserialize each of the individual
-  byte arrays into a single group element and output a new array
-  containing each of these elements.
-
-## API instantiation {#voprf-api}
+## Recommended ciphersuites {#voprf-ciph-recs}
 
-For the explicit signatures of each of the functions, refer to
-{{pp-functions}}.
+We recommend that the Issuer chooses one of the following ciphersuites
```

Comment:
```suggestion
The RECOMMENDED Issuer ciphersuites are as follows:
```

Reply:
Probably one of them should be MTI too, right?

Reply:
How is this usually specified?